index.js

```diff
@@ -225,6 +228,10 @@ function createMainWindow (done) {
 
     if (argv.debug) win.webContents.openDevTools()
     win.loadURL(INDEX)
+    // TODO: what's the way to see when the window is being refreshed?
+    win.on('refresh', function () {
+      server.sync.close()
```

Comment:
Is the idea that when a refresh happens sync is destroyed and then re-created?

Reply:
yeah, because it's a new sync session, it's mostly for development